packages/styles/scss/themes/expressive/_tokens.scss

```diff
@@ -116,13 +116,13 @@ $carbon-productive-type-tokens: (
 /// @group carbon-expressive
 $carbon-expressive-type-tokens: (
   body-short-01: (
-    font-size: carbon--type-scale(3),
+    font-size: carbon--type-scale(2),
```

Comment:
Don't think we should be changing the font size here as it will affect every component using these tokens.

Reply:
If we have to use `body-short-01` but we want 14px instead of 16px - there is a `@include use-carbon-productive-tokens();` mixin that we use.